presto-hive/src/test/java/com/facebook/presto/hive/TestShowStats.java

```diff
@@ -27,12 +29,22 @@ public TestShowStats()
         super(() -> createQueryRunner(ImmutableList.of()));
     }
 
-    @Test
-    public void testShowStats()
+    @BeforeClass
+    public void setUp()
     {
         assertUpdate("CREATE TABLE nation_partitioned(nationkey BIGINT, name VARCHAR, comment VARCHAR, regionkey BIGINT) WITH (partitioned_by = ARRAY['regionkey'])");
         assertUpdate("INSERT INTO nation_partitioned SELECT nationkey, name, comment, regionkey from tpch.tiny.nation", 25);
+    }
 
+    @AfterClass(alwaysRun = true)
```

Comment:
I don't think it is needed. The entire query runner will be killed after all the test cases from this class are ran

Reply:
Thanks, Do you mean tearDown() is not needed, or 'alwaysRun = true' is not needed ?

I was working the second diff, it is now pushed.

Reply:
I think teadDown is not needed